nuxeo-services/nuxeo-platform-dublincore/src/main/java/org/nuxeo/ecm/platform/dublincore/service/DublinCoreStorageServiceImpl.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * (C) Copyright 2006-2011 Nuxeo SA (http://nuxeo.com/) and others.
```

Comment:
Can you update 2011 to 2018 and remove SA for copyright ? You might put this change in a _Cleanup and format_ commit.